mypyc/irbuild/for_helpers.py

```diff
@@ -131,7 +132,41 @@ def for_loop_helper_with_index(builder: IRBuilder, index: Lvalue, expr: Expressi
     builder.activate_block(exit_block)
 
 
+def preallocate_space_helper(builder: IRBuilder,
```

Comment:
Let's try to make the name more descriptive. Maybe `sequence_from_generator_preallocate_helper`?